### PR TITLE
[build] Delete empty test directory and fix building empty project in worker mode

### PR DESCRIPTION
**Pull request by curtisman**
fluid-build changes:
- add emitting the error code on command error
- fix building empty project in worker mode

### CODE REVIEW

File: build-tools/packages/build-cli/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ $ npm install -g @fluid-tools/build-cli
 $ flub COMMAND
 running command...
 $ flub (--version|-V)
-@fluid-tools/build-cli/0.6.0
+@fluid-tools/build-cli/0.7.1
```
Comment (tylerbutler) [inline build-tools/packages/build-cli/README.md:20]: I think I figured out why the readmes weren't getting updated... #13689

**Review by tylerbutler — APPROVED** [review]
:shipit:

File: packages/framework/request-handler/src/test/requestHandlers.spec.ts [hunk 2]
```diff
@@ -92,7 +92,7 @@ describe("RequestParser", () => {
             const requestParser = RequestParser.create({ url: "/objectId/route", headers: { wait: true } });
             const response = await rootDataStoreRequestHandler(requestParser, runtime);
             assert.equal(response?.status, 200);
-            assert.equal(response.value.route, "/route");
+            assert.equal(response?.value.route, "/route");
```
Comment (tylerbutler) [inline packages/framework/request-handler/src/test/requestHandlers.spec.ts:95]: I think this is an unrelated change?

Comment (curtisman) [reply packages/framework/request-handler/src/test/requestHandlers.spec.ts:95]: A little bit,  but I keep on hitting compile error locally. So I just fixed that.  Not sure why it does fail in the CI
